Adds comment about data sampling time + general enhancements (#2639)
elastic/docs-content: deploy-manage/cloud-organization/billing.md

# Billing

> Elastic charges a recurring fee for using our offerings on {{ecloud}}. In this section, you'll learn how to about the dimensions used to calculate your bill, how to monitor account usage, how to manage billing, and more.

Reviewer: did you mean this?

```suggestion
Elastic charges a recurring fee for using our offerings on {{ecloud}}. In this section, you'll learn how dimensions are used to calculate your bill, how to monitor account usage, how to manage billing, and more.
```

Author: @bmorelli25 Sorry for the confusion, I didn't make these changes. I accidentally pulled these changes into my PR when fixing a merge conflict. I'm trying to revert it now
